website/source/docs/providers/azurerm/r/express_route_circuit.html.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_express_route_circuit"
+sidebar_current: "docs-azurerm-resource-express-route-circuit"
+description: |-
+  Creates an ExpressRoute circuit.
+---
+
+# azurerm\_express\_route\_circuit
+
+Creates an ExpressRoute circuit.
+
+## Example Usage
+
+```hcl
+resource "azurerm_express_route_circuit" "test" {
+  name                     = "expressRoute1"
+  resource_group_name      = "${azurerm_resource_group.test.name}"
+  location                 = "West US"
+  service_provider_name    = "Equinix"
+  peering_location         = "Silicon Valley"
+  bandwidth_in_mbps        = 50
+  sku_tier                 = "Standard"
+  sku_family               = "MeteredData"
+  allow_classic_operations = false
+
+  tags {
+    environment = "Production"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the ExpressRoute circuit. Changing this forces a
+    new resource to be created.
+
+* `resource_group_name` - (Required) The name of the resource group in which to
+    create the namespace. Changing this forces a new resource to be created.
+
+* `location` - (Required) Specifies the supported Azure location where the resource exists.
+    Changing this forces a new resource to be created.
+
+* `service_provider_name` - (Required) The name of the ExpressRoute Service Provider.
+
+* `peering_location` - (Required) The name of the peering location and not the ARM resource location.
+
+* `bandwidth_in_mbps` - (Required) The bandwidth in Mbps of the circuit being created.
+
+* `sku_tier` - (Optional) Chosen SKU Tier of ExpressRoute circuit. Choose from Premium or Standard SKU tiers. 
+    The default value is "Standard".
+
+* `sku_family` - (Optional) Chosen SKU family of ExpressRoute circuit. 
+    Choose from MeteredData or UnlimitedData SKU families. The default value is "MeteredData".
```

Comment:
These two parameters in ARM templates on the portal are optional. Actually I'm not sure should TF map that or not...